common/hooks/useCollectionDialog.ts

```diff
@@ -3,7 +3,7 @@
 import { useState, useCallback, useEffect } from "react";
 import { useDialogContext } from "common/hooks/useDialogContext";
 import { useMyCollections } from "common/hooks/useMyCollections";
-import { Option } from "common/types/types";
+import { Collection, Option } from "common/types/types";
```

Comment:
Suggestion, add a cached  state for the initial selected options, if the user closes the dialog something like  ` const [initialSelectedOptions, setInitialSelectedOptions] = useState<string[]>([]);`  and in the handleCancel add the   `setSelectedOptions(initialSelectedOptions); `  this is to its original state if the dialog is canceled

Reply:
Currently, when canceling the dialog it does get back to it's original state. If you see room for improvement on how this is implemented, we can perhaps do it in a different PR?